src/Content/__tests__/RichContent.test.js

```diff
@@ -0,0 +1,36 @@
+import { createElement } from 'react';
+import { RichContent } from '..';
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+test('Renders raw HTML', () => {
+    let wrapper = mount(
+        <RichContent sanitizedRawHtml="<h1 id='o-no'>Raw!!!</h1>" />
+    );
+    expect(wrapper.html()).toEqual(
+        '<span class="peregrine-raw-html"><h1 id="o-no">Raw!!!</h1></span>'
+    );
+    wrapper = mount(
```

Comment:
This should be a separate test case. It's important that, if this one assertion fails, the description of the test failing is clear. Right now, if we have a bug that favors the `null` children, the name of the test failing would be `Renders raw HTML`, which isn't descriptive of what we're testing.

Reply:
Good point; I've changed it.